README.md

```diff
@@ -74,89 +88,216 @@ spec:
     registryPoll:
       interval: 5m
 ```
-Replace value of `image` field with your catalog source image. Don't forget to specify desired platform.
+Replace value of `image` field with your catalog source image. Don't forget to specify the desired platform.
 
 5. Deploy Che using chectl:
 ```sh
 $ chectl server:deploy --installer=olm --platform=${platform} -n ${che-namespace} --catalog-source-yaml ${path_to_custom_catalog_source_yaml} --olm-channel=nightly --package-manifest-name=eclipse-che-preview-${platform}
 ```
 
-### OpenShift oAuth
+## Deploy che-operator using scripts
+
+**IMPORTANT! Cluster Admin privileges are required**
+
+```bash
+./deploy.sh $namespace
+```
+
+The script will create sa, role, role binding, operator deployment, CRD and CR.
+
+Wait until Che deployment is scaled to 1 and Che route is created.
+
+When on pure k8s, make sure you provide a global ingress domain in `deploy/crds/org_v1_che_cr.yaml` for example:
+
+```bash
+  k8s:
+    ingressDomain: '192.168.99.101.nip.io'
+```
+
+### Edit checluster CR using command line interface(terminal)
+
+Any Kubernetes object you can modify using the UI(for example Openshift console).
+But also you can do the same using terminal.
+You can edit checluster CR object using command line editor: 
+
+```bash
+$ oc edit checluster ${che-cluster-name} -n ${namespace}
+```
+
+Where ${che-cluster-name} is a custom resource name, by default 'eclipse-che'.
+
+Also you can modify the CheCluster using the `kubectl patch`. For example:
+
+```bash
+$ kubectl patch checluster/eclipse-che -n ${eclipse-che-namespace} --type=merge -p '{"spec":{"auth":{"openShiftoAuth": false}}}'
+```
+
+### Update checluster using chectl
 
-Bear in mind that che-operator service account needs to have cluster admin privileges so that the operator can create oauthclient at a cluster scope.
+You can update Che configuration using: command `chectl server:update` and flag `--cr-patch`
+
+## OpenShift oAuth
+
+OpenShift clusters includes a built-in OAuth server. che-operator supports this authentication way.
+There is CR property 'openShiftoAuth' to enable/disable this feature. It's enabled by default.
+
+> INFO: If you are [using scripts to deploy che-operator](#deploy_che_operator_using_scripts), then  bear in mind that che-operator service account needs to have cluster admin privileges so that the operator can create oauthclient at a cluster scope.
```

Comment:
```suggestion
> INFO: If you are [using scripts to deploy che-operator](#deploy_che_operator_using_scripts), note that the che-operator service account needs to have cluster-admin privileges so that the operator can create OAuth client at a cluster scope.
```